cloud_foundry_api/assets/configuration/spec.yaml

```
@@ -0,0 +1,76 @@
+name: Cloud Foundry API
+files:
+- name: cloud_foundry_api.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
```

```suggestion
    - template: init_config/default
    - template: init_config/http
```

For consistency as this integration uses the http wrapper